packages/@sanity/code-input/src/CodeInput.js

```diff
@@ -64,6 +67,31 @@ export default class Code extends React.PureComponent {
     ]))
   }
 
+  handleToggleSelectLine = line => {
+    const {type, onChange} = this.props
+    const path = ['highlightedLines']
+
+    const highlightedLines = xor(this.props.value.highlightedLines, [line]).sort(compareNumbers)
```

Comment:
For readability, could you destruct `this.props.value` as you do with `type` and `onChange`?